lib/UnitFactory.js

```diff
@@ -11,6 +11,50 @@ UnitFactory.createUnit = function(unitType, location) {
     var stats;
     // MAKE SURE YOU ADD TO UnitType and Player Factor too!
     switch(unitType) {
+        case UnitType.BULL:
+            stats = {
+                "name": "bull",
+                "description": "Bulls are big. Don't make them angry",
+                "hint": "Don't attack them, and they won't attack you.",
+                "team": Team.CREEP,
+                "type": unitType,
+                "maxHp": 30,
+                "location": location,
+                "movementDelay": 1,
+                "meleeAttackPower": 5,
+                "meleeAttackDelay": 1,
+                "magicAttackPower": 0,
+                "magicAttackDelay": 0,
+                "rangedAttackPower": 0,
+                "rangedAttackDelay": 0,
+                "sensorRadiusSquared": 64,
```

Comment:
gotcha. I also did this in the behavior of the bull by saying, if the bull has full health, don't attack. so it will only attack if attacked, but it should be able to respond to a range attack too